cmd/traffic/cmd/manager/service.go

```diff
@@ -500,6 +562,14 @@ func (s *service) CreateIntercept(ctx context.Context, ciReq *rpc.CreateIntercep
 		}
 	}
 
+	SetGauge(s.metrics.InterceptActiveStatusGauge, client.Name, client.InstallId, spec.Name, 1)
+
+	if spec.Mechanism == "tcp" {
+		IncrementCounter(s.metrics.InterceptGlobalCounter, client.Name, client.InstallId)
+	} else {
+		IncrementCounter(s.metrics.InterceptPersonalCounter, client.Name, client.InstallId)
+	}
```

Comment:
This distinction is not valid i OSS. A non "tcp" mechanism doesn't mean that it's personal (because there's no such thing).